examples/add_filetype_object_from_csv_v2.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import csv
+from pymisp import ExpandedPyMISP, MISPObject
```

Comment:
You can (should) use `PyMISP` and not `ExpandedPyMISP`. They're the same and `ExpandedPyMISP` is deprecated.